src/amo/components/PermissionsCard/styles.scss

```diff
@@ -0,0 +1,29 @@
+.PermissionCard-learn-more {
+  margin: 12px auto 0;
+  width: 100%;
+
+  .Icon {
+    margin-left: 10px;
```

Comment:
To handle right-to-left locales, which will reverse the icon/text order, you'll need to use [our mixin](https://github.com/mozilla/addons-frontend/blob/master/src/core/css/inc/mixins.scss#L144) like: 

```css
@import "~core/css/inc/mixins";

.Icon {
  @include margin-start(12px);
}
```